subsys/mgmt/mcumgr/lib/cmd/shell_mgmt/src/shell_mgmt.c

```diff
@@ -10,16 +10,37 @@
 #include "mgmt/mgmt.h"
 #include "cborattr/cborattr.h"
 #include "shell_mgmt/shell_mgmt.h"
-#include "shell_mgmt/shell_mgmt_impl.h"
 #include "shell_mgmt/shell_mgmt_config.h"
+#include <shell/shell_dummy.h>
+#include <shell/shell_dummy.h>
```

Comment:
Duplicate include

Reply:
It is because this must be included for sure, we need the redundancy in case the firs line doesn't work.

Reply:
just kidding

Reply:
again, @de-nordic must fix this using a rebase, which will dismiss all approvals, and then he must wait on approvals again,

Reply:
So be it, we can't merge this with duplicate includes. @de-nordic please fix it and we'll re-approve and merge

Reply:
> It is because this must be included for sure, we need the redundancy in case the first line doesn't work.

Never trust the preprocessor.